.github/workflows/deploy.yml

```
@@ -0,0 +1,32 @@
+name: git push into another repo to deploy to vercel
+
+on:
+  push:
+    branches: [main, feat/MG-15]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    container: pandoc/latex
+    steps:
+      - uses: actions/checkout@v2
```

actionlint (1.7.7)

the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)